packages/compass-components/src/components/placeholder.tsx

```diff
@@ -21,8 +21,8 @@ const move = keyframes({
 });
 
 const placeholder = css({
-  '--gradient-start': uiColors.gray.light3,
-  '--gradient-end': 'rgba(235, 241, 239, 1)',
+  '--gradient-start': uiColors.gray.light2,
```

Comment:
There are no designs for these things so I just picked some that have more contrast.